src/server/nyplApiClient/index.ts

```diff
@@ -58,6 +59,16 @@ const nyplApiClient = async ({
       oauth_url: appConfig.urls.tokenUrl,
     })
     CACHE.clients[clientCacheKey] = nyplApiClient
+    const get = nyplApiClient.get.bind(nyplApiClient)
```

Comment:
This is the first time seeing this update. You can get around `bind` if you use `apply`:
```
const get = nyplApiClient.get;

nyplApiClient.get = async function (...args) {
  logger.info(`GET ...`);
  return get.apply(this.args);
}
```

It's minor but you don't need to create a function with `bind`  and can call the original function directly while overriding it.